docs/reference-guides/cluster-configuration/rancher-server-configuration/k3s-cluster-configuration.md

```diff
@@ -98,6 +98,14 @@ Select the domain for the cluster. The default is `cluster.local`.
 
 Option to change the range of ports that can be used for [NodePort services](https://kubernetes.io/docs/concepts/services-networking/service/#nodeport). The default is `30000-32767`.
 
+#### Truncate Hostnames
+
+Option to truncate hostnames, for NetBIOS compatibility. Although Kubernetes allows hostnames up to 63 characters in length, Windows systems have a hard limit of 15 characters. 
+
+You can only set this field during the initial creation of the cluster. You can't enable or disable the limit after cluster creation.
+
+The default hostname limit for this field is 15 characters. Once enabled, you can optionally change the hostname limit to a value less than 15 characters.
```

Comment:
Suggesting to remove this - https://github.com/rancher/rancher-docs/issues/607#issuecomment-1577133848